src/App.vue

```diff
@@ -41,19 +65,21 @@ export default defineComponent({
       }
     }
   },
-  async mounted() {
-    this.loader = await loadingController
-      .create({
-        message: this.$t("Click the backdrop to dismiss."),
-        translucent: true,
-        backdropDismiss: true
-      });
+  mounted() {
+    emitter.on('timeZoneDifferent', this.timeZoneDifferentAlert);
```

Comment:
This needs to be emitted from userprofile action
https://github.com/hotwax/preorder/blob/5f56d984dbcf531c116b81035ad70abecc18bea3/src/store/modules/user/actions.ts#L58

Reply:
Yes sir, we are emitting this from user getprofile action.
https://github.com/hotwax/ionic-sdk/blob/main/src/store/modules/user/actions.ts#L58